ocis-pkg/shared/healthchecklist.go

```diff
@@ -0,0 +1,29 @@
+package shared
```

Comment:
Can we have a more speaking package then just "shared"? Maybe "health" or something like that?

Reply:
Fine by me. I'm just generally not a fan of those generic packages like "util" or "shared" but you didn't introduce it so no blame on you. ;)